### DISCUSSION TITLE
NextJS getInitialProps || getStatisProps with accountsGraphql.getUser()

### DISCUSSION THREAD

**Discussion by edw19** [accounts-js/accounts#1093] [General]
I'm trying to manage my app's authentication in nextjs and I'm having trouble knowing which user is authenticated because when using accountsGraphql.getUser () on the server side it gives me a localStorage error. Does anyone know the correct way to implement it in nextjs?

**Comment by pradel** [comment]
@edw19 the best way to do this is to replace the localStorage config with for example https://github.com/js-cookie/js-cookie
